core/domain/platform_parameter_registry.py

```diff
@@ -443,3 +443,22 @@ def _to_platform_parameter_model(
     platform_parameter_domain.DataTypes.NUMBER,
     default=0.2
 )
+
+# Ensure that SYSTEM_EMAIL_ADDRESS and ADMIN_EMAIL_ADDRESS are both valid and
+# correspond to owners of the app before setting this to True. If
+# SYSTEM_EMAIL_ADDRESS is not that of an app owner, email messages from this
+# address cannot be sent. If True then emails can be sent to any user.
+Registry.create_platform_parameter(
+    ParamName.CAN_SEND_EMAILS,
+    (
+        'Whether the application can send emails.'
+        'This should be set to True only in production environment.'
```

Comment:
Is the intent to change the default value here for production deployments or to change the value inside the platform parameter dashboard? I'd assume the latter since that's the best way for us to ensure the value is correctly on for prod instances of the app.

In either case, please clarify the description here so that the reader knows what needs to be done to actually enable the parameter.